webapp/app/components/Message/Message.component.js

```diff
@@ -0,0 +1,55 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+import moment from 'moment';
+import { ListItem, ListItemText } from 'material-ui/List';
+
+import { Author, Data, MessageContent } from './Message.styles';
+import { UserAvatar } from '../UserAvatar/UserAvatar.component';
+
+
+export class Message extends PureComponent {
+  static propTypes = {
+    message: PropTypes.instanceOf(Map),
+  };
+
+  get time() {
```

Comment:
sprobuj uzyc moze humanize z moment

Reply:
Z tego co widzę w dokumentacji https://momentjs.com/docs/#/durations/humanize/
`humanize` można użyć do podania informacji jaki czas temu coś się wydarzyło, czy o to Tobie chodziło ?
Tutaj akurat podaje konkretną godzinę, informacja w stylu `5 minut temu` miało by tą wadę, że trzeba by to było w jakiś sensowny sposób odświeżać.